src/pages/BuyGMX/BuyGMX.tsx

```diff
@@ -0,0 +1,256 @@
+import React, { useCallback } from "react";
+import Footer from "components/Footer/Footer";
+import "./BuyGMX.css";
+import { useWeb3React } from "@web3-react/core";
+import { Trans, t } from "@lingui/macro";
+import Button from "components/Common/Button";
+import { ARBITRUM, AVALANCHE, getChainName, getConstant } from "config/chains";
+import { switchNetwork } from "lib/wallets";
+import { useChainId } from "lib/chains";
+import Card from "components/Common/Card";
+import { importImage } from "lib/legacy";
+import ExternalLink from "components/ExternalLink/ExternalLink";
+
+import Banxa from "img/ic_banxa.svg";
+import Uniswap from "img/ic_uni_arbitrum.svg";
+import Traderjoe from "img/ic_traderjoe.png";
+import Bungee from "img/ic_bungee.png";
+import O3 from "img/ic_o3.png";
+import Binance from "img/ic_binance.svg";
+import ohmArbitrum from "img/ic_olympus_arbitrum.svg";
+import { CENTRALISED_EXCHANGES, DECENTRALISED_AGGRIGATORS, EXTERNAL_LINKS, TRANSFER_EXCHANGES } from "./constants";
+
+export default function BuyGMX() {
+  const { chainId } = useChainId();
+  const isArbitrum = chainId === ARBITRUM;
+  const { active } = useWeb3React();
+  const nativeTokenSymbol = getConstant(chainId, "nativeTokenSymbol");
+
+  const onNetworkSelect = useCallback(
+    (value) => {
+      if (value === chainId) {
+        return;
+      }
+      return switchNetwork(value, active);
+    },
+    [chainId, active]
+  );
+
+  return (
+    <div className="BuyGMXGLP default-container page-layout">
+      <div className="BuyGMXGLP-container">
+        <div className="section-title-block">
+          <div className="section-title-content">
+            <div className="Page-title">
+              <Trans>Buy GMX on {getChainName(chainId)}</Trans>
+            </div>
+            <div className="Page-description">
+              <Trans>Choose to buy from decentralized or centralized exchanges.</Trans>
+              <br />
+              <Trans>
+                To purchase GMX on the {isArbitrum ? "Avalanche" : "Arbitrum"} blockchain, please{" "}
+                <span onClick={() => onNetworkSelect(isArbitrum ? AVALANCHE : ARBITRUM)}>change your network</span>.
+              </Trans>
+            </div>
+          </div>
+        </div>
+        <div className="cards-row">
+          <DecentralisedExchanges chainId={chainId} />
+          <CentralisedExchanges chainId={chainId} />
+        </div>
+
+        {isArbitrum ? (
+          <div className="section-title-block mt-top">
```

Comment:
As a point of discussion:
I think we need to make some kind of convention on the names of css classes

Because right now some classes are named with component prefix, some are not, and there are also different global classes.

This can be confusing, also we have no isolation and `section-title-block` class can affect styles of nested components

I think it would be good to name all local component classes with the prefix (BuyGMX)

Without the prefix to use only global classes (by the way, it would be nice to systematize them somehow, but that's a separate topic)